pipelines/matrix/src/matrix/pipelines/integration/pipeline.py

```diff
@@ -29,7 +29,17 @@ def _create_integration_pipeline(
                         # during node execution time, otherwise we could infer this based on
                         # the transformer.
                         **({"nodes_df": f"ingestion.int.{source}.nodes"} if has_nodes else {}),
-                        **({"edges_df": f"ingestion.int.{source}.edges"} if has_edges else {}),
+                        **(
+                            {"edges_df": f"ingestion.int.{source}.edges"} if (has_edges and not is_ground_truth) else {}
+                        ),
+                        **(
+                            {
+                                "positive_edges_df": f"ingestion.int.{source}.positive.edges@spark",
+                                "negative_edges_df": f"ingestion.int.{source}.negative.edges@spark",
+                            }
+                            if (has_edges and is_ground_truth)
```

Comment:
Should it be `not is_ground_truth` here?
```suggestion
                            if (has_edges and not is_ground_truth)
```

Reply:
I'm wondering if it would be useful to store `has_edges and not is_ground_truth` in a variable to avoid this kind of problem, but I'm probably overthinking it